api/konfig/builtinpluginconsts/namespace.go

```diff
@@ -6,14 +6,12 @@ package builtinpluginconsts
 const (
 	namespaceFieldSpecs = `
 namespace:
-- path: metadata/namespace
-  create: true
 - path: metadata/name
   kind: Namespace
   create: true
-- path: subjects
+- path: subjects/namespace
```

Comment:
These are also stripped. Should we fix them (as this is doing) or just remove them?

Reply:
Is there any purpose of keeping them in here? I see why we need to fix them for user provided fieldspecs but I'm not sure I understand the purpose of keeping invalid values in our builtin fieldspecs.

Reply:
It would just be to show that the transformer affects these fields, I guess. In theory we could use it as a signal of whether or not we should (manually) modify them, but in practice that could be a breaking change for people who have existing manually configured transformers. I'm definitely fine with removing them.

Reply:
I have a slight preference for  removing them, but don't feel too strongly either way.

Reply:
removed